src/error-handling/converting-error-types.md

```diff
@@ -19,15 +19,17 @@ The `From::from` call here means we attempt to convert the error type to the
 type returned by the function:
 
 ```rust,editable
-use std::{fs, io};
+use std::{fs::{self, File}, io};
```

Comment:
I know it's a bit of taste, but I really dislike the nested import syntax. It feels very over-engineered to me and I would rather not present it to brand new Rust developers :-)

I'm also not a fan of the `self` — I just like multi-line imports with a nice and consistent hiearchy as you go down the list. Perhaps you restructured this to make the code fit on a slide?

Reply:
Changed to remove nested braces.